Zend/zend_API.h

```diff
@@ -1159,6 +1159,7 @@ ZEND_API ZEND_COLD void ZEND_FASTCALL zend_wrong_callback_error(int num, char *e
 			if (!(_flags & ZEND_PARSE_PARAMS_QUIET)) { \
 				if (_error_code == ZPP_ERROR_WRONG_CALLBACK) { \
 					zend_wrong_callback_error(_i, _error); \
+					efree(_error); \
```

Comment:
This code is size-sensitive, so it would be better not to move efree() out.

Reply:
ok, but my computer is out of power...
I will fix it tomorrow